composer.json

```diff
@@ -16,10 +16,10 @@
     },
     "require": {
         "php": "^8.0",
-        "nesbot/carbon": "^2.38",
+        "nesbot/carbon": "^3.0",
```

Comment:
I'm actually using this PR on my project (otherwise I can't use Symfony 7) and it works with no issue, but I understand you want to separate the upgrades 👍
I'm not at home right now, but will remove the Carbon upgrade part once I can, and make a separate PR on top of that one :)